github/event_types.go

```diff
@@ -480,19 +480,24 @@ type PublicEvent struct {
 // GitHub API docs: https://developer.github.com/v3/activity/events/types/#pullrequestevent
 type PullRequestEvent struct {
 	// Action is the action that was performed. Possible values are: "assigned",
-	// "unassigned", "labeled", "unlabeled", "opened", "closed", or "reopened",
-	// "synchronize", "edited". If the action is "closed" and the merged key is false,
+	// "unassigned","review_requested","review_request_removed" "labeled", "unlabeled",
+	// "opened", "closed", or "reopened","synchronize", "edited".
+	// If the action is "closed" and the merged key is false,
 	// the pull request was closed with unmerged commits. If the action is "closed"
 	// and the merged key is true, the pull request was merged.
 	Action      *string      `json:"action,omitempty"`
 	Number      *int         `json:"number,omitempty"`
 	PullRequest *PullRequest `json:"pull_request,omitempty"`
 
 	// The following fields are only populated by Webhook events.
-	Changes      *EditChange   `json:"changes,omitempty"`
-	Repo         *Repository   `json:"repository,omitempty"`
-	Sender       *User         `json:"sender,omitempty"`
-	Installation *Installation `json:"installation,omitempty"`
+	Changes *EditChange `json:"changes,omitempty"`
+	Repo    *Repository `json:"repository,omitempty"`
+	Sender  *User       `json:"sender,omitempty"`
+	// According to Github API Docs,
+	// Deliveries for review_requested and review_request_removed events
+	// will have an additional field called requested_reviewers.
+	RequestedReviewers []*User       `json:"requested_reviewers,omitempty"`
```

Comment:
Get rid of "According to Github API Docs", and the rest can be simplified. We should say "RequestedReviewers" rather than "requested_reviewers", since the Go field is named `RequestedReviewers`. How about just this:

```Go
RequestedReviewers []*User `json:"requested_reviewers,omitempty"` // For "review_requested", "review_request_removed" events.
```

Reply:
Also, let's move this field up by two, so it's under `RequestedReviewers` and before the more generic `Repo`, `Sender`. It feels out of place here.